.rubocop.yml

```diff
@@ -147,3 +147,8 @@ Rails/HasManyOrHasOneDependent:
     - "app/models/saved_claim/education_benefits.rb"
     - "app/models/saved_claim.rb"
     - "app/models/terms_and_conditions.rb"
+
+Style/BlockDelimiters:
+  Exclude:
+    - "spec/factories/*.rb"
+    - "modules/*/spec/factories/*.rb"
```

Comment:
I think a concise "why"-style comment is warranted here. It could save a future engineer who is cleaning up exemptions some time.

Reply:
Fair enough. I think we should probably annotate *everything* in the rubocop.yml. And use an RFC doc to clarify it so there's a mechanism for changing that's not super-adHoc